algorithmic_efficiency/workloads/wmt/workload.py

```diff
@@ -56,7 +56,7 @@ def num_validation_examples(self) -> int:
   @property
   def num_test_examples(self) -> int:
     # wmt14_translate/de-en 'test' split size.
-    return 3003
+    return 3000
```

Comment:
Why are we not using the full test set anymore? Since padding has been implemented, I think this fix is not necessary anymore.